core/src/main/java/com/graphhopper/routing/lm/LandmarkStorage.java

```diff
@@ -108,8 +108,8 @@ public String toString() {
             }
         };
 
-        // later make edge base working! Not really necessary as when adding turn costs while routing we can still
-        // use the node based traversal as this is a smaller weight approximation
+        // Edge based is not really necessary because when adding turn costs while routing we can still
+        // use the node based traversal as this is a smaller weight approximation and will still produce correct results
```

Comment:
Not sure if I get your comment right. You are stating that during the creation of LM the traversal mode is not relevant, only during routing, as we only forbid certain turns?